vscode/src/chat/chat-view/SidebarChatProvider.ts

```diff
@@ -145,7 +145,7 @@ export class SidebarChatProvider extends MessageProvider implements vscode.Webvi
                 void openExternalLinks(message.value)
                 break
             case 'openFile':
-                await openFilePath(message.filePath, this.webviewPanel?.viewColumn)
+                await openFilePath(message.filePath, undefined, this.webviewPanel?.viewColumn)
```

Comment:
```suggestion
                await openFilePath(message.filePath, message.uri, this.webviewPanel?.viewColumn)
```

`message.uri` should also work here?

Reply:
This was intentional, as I didn't want to interfere with any legacy behavior. We fall back to using the file path if the uri is undefined.